### DISCUSSION TITLE
Update from Launcher

### DISCUSSION THREAD

**Discussion by Cyanidexhq** [rfvgyhn/min-ed-launcher#128] [Q&A] [answered]
Hi o7 

First of all, I want to say thank you for min-ed-launcher because my launcher stuck on "working" screen but seems like min-ed-launcher is really fast to get into the game. 

The problem, my epic account updates the game from launcher that I can not see the "play button". How can I update my epic game while using min-ed-launcher ? 

and I know I can login to my epic account by using my steam game (I read the readme file) but I want to launch 2 game in the same time. 

Thanks.

"[17:27:02 INF] Available Products:
Elite Dangerous: Horizons FORC-FDEV-DO-38-IN-40 Requires Update
Legacy Horizons FORC-FDEV-D-1013 Up to Date
Legacy Elite Dangerous FORC-FDEV-D-1010 Up to Date
Elite Dangerous: Arena FORC-FDEV-D-1012 Up to Date
Single Player Combat Training COMBAT_TUTORIAL_DEMO Not Installed
Select a product to launch (default=1):

Legacy Horizons
Legacy Elite Dangerous
Elite Dangerous: Arena
Exit
Product:
"
My epic account has no odyssey

**Comment by rfvgyhn** [comment, marked as answer]
If I understand correctly, you want to have min-ed-launcher handle the updates instead of Epic, right? If so, you can update your [settings]() file and include the Horizons SKU in `forceUpdate`.

 `"forceUpdate": "FORC-FDEV-DO-38-IN-40"`

**Reply by Cyanidexhq** [reply]
Yep it worked ! thanks again o7
